addons/web/views/report_templates.xml

```diff
@@ -353,13 +353,13 @@
     <template id="external_layout_boxed">
         <div t-attf-class="header o_company_#{company.id}_layout" t-att-style="report_header_style">
             <div class="o_boxed_header">
-            <div class="row mb8">
+            <div class="row">
```

Comment:
since these view with these classes exist for a long time and are rather central of report, I think they have a high change to be customized, so that the chance of breaking XPath in customization exist (even if it's a bad idea to rely on classes for XPATH). So if possible I think it would be safer to have a CSS only solution.